src/main/java/org/sagebionetworks/web/client/widget/entity/EntityBadge.java

```diff
@@ -291,22 +293,37 @@ public void onAddToDownloadList() {
 			globalAppState.getPlaceChanger().goTo(new LoginPlace(LoginPlace.LOGIN_TOKEN));
 			return;
 		}
-		// TODO: add special popup to report how many items are in the current download list, and link to
```

Comment:
Adding a single file to the DL from the files browser (each row is an EntityBadge).  Use new service if in alpha mode.